apps/prairielearn/src/pages/partials/navTabs/instructorQuestion.ejs

```diff
@@ -21,4 +21,10 @@
     urlSuffix:      '/question/'+question.id+'/statistics',
     iconClasses:    'fas fa-chart-bar',
     tabLabel:       'Statistics'}) %>
+<%- include('navTab', {
+    activeSubPage:  'issues',
+    urlSuffix:      '/course_admin/issues?q=is%3Aopen+qid%3A'+encodeURIComponent(question.qid),
+    iconClasses:    'fas fa-bug',
+    tabLabel:       'Issues',
+    htmlSuffix: include("../issueBadge", {count: open_issue_count, suppressLink: true, className: 'ml-2'})}) %>
```

Comment:
Nit, idiosyncratic formatting from this page (we'll get this as TypeScript soon hopefully):

```suggestion
    tabLabel:       'Issues',
    htmlSuffix:     include("../issueBadge", {count: open_issue_count, suppressLink: true, className: 'ml-2'})}) %>
```